go.mod

```diff
@@ -37,6 +37,7 @@ require (
 	github.com/nishanths/exhaustive v0.0.0-20200525081945-8e46705b6132
 	github.com/pkg/errors v0.9.1
 	github.com/ryancurrah/gomodguard v1.1.0
+	github.com/ryanrolds/sqlclosecheck v0.3.0-rc
```

Comment:
could we pick a release version not a rc version?

Reply:
Yup. Set to v0.3.0 now.